README.md

```diff
@@ -6,7 +6,91 @@ For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/maste
+
+These projects are located in the following repositories:
+
+* https://github.com/nodejs/build — administration delegated to the Build Working Group
+* https://github.com/nodejs/citgm — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/CTC — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/docker-node — administration delegated to the Docker Working Group
+* https://github.com/nodejs/http-parser — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/LTS — administration delegated to the LTS Working Group
+* https://github.com/nodejs/node-eps — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/nodejs-dist-indexer — administration delegated to the Build Working Group
+* https://github.com/nodejs/nodejs-latest-linker — administration delegated to the Build Working Group
+* https://github.com/nodejs/nodejs-nightly-builder — administration delegated to the Build Working Group
+* https://github.com/nodejs/nodejs.org — administration delegated to the Website Working Group
+* https://github.com/nodejs/secrets — _private_, administration delegated to the Build Working Group
+* https://github.com/libuv/libuv — currently under Incubation
+
+### Adjacent Projects
+
+The TSC is responsible for a number of projects that are not strictly required to plan, test, build, document and ship Node.js releases. Projects that are _adjacent_ are either created from within the technical organization managed by the TSC or are adopted into that organization from outside.
+
+In the case of adopting existing projects, once the TSC has decided that adoption appropriate, it should seek agreement from the Node.js Foundation Board for such adoption as it impacts on the scope of technical activities of the Foundation.
+
+Current adjacent projects that are within the TSC's scope of responsibility can be found at the following locations:
+
+* https://github.com/nodejs/api — administration delegated to the API Working Group
+* https://github.com/nodejs/benchmarking — administration delegated to the Benchmarking Working Group
+* https://github.com/nodejs/code-and-learn
+* https://github.com/nodejs/diagnostics — administration delegated to the Diagnostics Working Group
+* https://github.com/nodejs/docs — administration delegated to the Documentation Working Group
+* https://github.com/nodejs/education
+* https://github.com/nodejs/email — administration delegated to the Build Working Group
+* https://github.com/nodejs/evangelism — administration delegated to the Evangelism Working Group
+* https://github.com/nodejs/github-bot — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/help — administration delegated to the Core Technical Committee
```

Comment:
@williamkapke is this based on how it's managed? I haven't paid close attention to who actively administers it.